.github/workflows/check.yaml

```diff
@@ -75,6 +75,11 @@ jobs:
           juju version
           juju controllers | grep Version -A 1 | awk '{print $9}'
 
+      - name: Show charmcraft information
+        run: |
+          snap list charmcraft
+          charmcraft --version
```

Comment:
charmcraft v3 is installed here, but I don't know what installs it.  Does anyone know?

Reply:
Ah it's installed by charmed-kubernetes/actions-operator https://github.com/charmed-kubernetes/actions-operator/blob/5c6377ed695d52b8a1693f07b7d641e245269123/action.yaml#L36

Reply:
I suppose this will be removed after testing?

Reply:
Perhaps - maybe it would be worth keeping for future reference?  I see we have a similar one above for juju information.

Reply:
Ah it broke it, because charmcraft version flag changed between v2 and v3.  Ok I'll remove it.